_data/simple-icons.json

```diff
@@ -7884,7 +7888,7 @@
         {
             "title": "Safari",
             "hex": "000000",
-            "source": "https://images.techhive.com/images/article/2014/11/safari-favorites-100530680-large.jpg"
+            "source": "https://es.wikipedia.org/wiki/Archivo:Safari_browser_logo.svg"
```

Comment:
Might be better to link to wikimedia instead:
https://commons.wikimedia.org/wiki/File:Safari_browser_logo.svg